spark/src/main/scala/geotrellis/spark/tiling/ZoomedLayoutScheme.scala

```diff
@@ -116,13 +116,15 @@ class ZoomedLayoutScheme(val crs: CRS, val tileSize: Int, resolutionThreshold: D
 
   def zoomOut(level: LayoutLevel) = {
     val layout = level.layout
+    val newZoom = level.zoom - 1
+    val newSize = math.pow(2, newZoom).toInt
```

Comment:
Does this change allow to correctly pyramid a floating layer into a zoomed layout scheme? I think so, but there should be a small unit test for that.

If all of that works out fine I'm not sure there is any reason to change `FloatingLayoutScheme` as its primary purpose it to produce a layer with cell width matching the source rasters.